drivers/sx127x/sx127x.c

```diff
@@ -140,12 +135,6 @@ int sx127x_init(sx127x_t *dev)
         return -SX127X_ERR_NODEV;
     }
 
-    /* Check if the reset pin is defined */
-    if (dev->params.reset_pin == GPIO_UNDEF) {
-        DEBUG("[sx127x] error: No reset pin defined.\n");
-        return -SX127X_ERR_GPIOS;
-    }
-
     _init_timers(dev);
 
     /* reset pin should be left floating during POR */
```

Comment:
Reset pin is still being initialized here.

Reply:
Ah yes, should be fixed now.